modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -207,4 +209,21 @@ module LocaleModelHelpSetup {
     dst.GPU = new unmanaged GPULocale(1:chpl_sublocID_t, dst);
     chpl_task_setSubloc(origSubloc);
   }
+
+  proc helpSetupLocaleGPU(dst: borrowed LocaleModel, numSublocales: int,
```

Comment:
Is there a strong reason why the `numSublocales` and `local_name` arguments are in the reverse order here, compared to `helpSetupLocaleAPU()` above?  If not, could they be made the same just to reduce the number of differences between locale models?